### PR TITLE
Change fork icon's color in readme

### LINKED ISSUE #302: [DOCS] fork image not visible in dark mode
### Description

After the click on the in the  line 1 in screenshot it looks empty which is not looking good so we should make some changes to make it look clear

### Screenshots

Screenshot 2021-09-23 at 4 04 50 PM

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by riadh26**
Fixes #302 
Fixed my previous pull request #312 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/317"></a>

### CODE REVIEW

**Review by niteshseram — APPROVED** [review]
LGTM 👍. BTW you could have just committed your changes in your last PR 😃

**Review by rohan-kulkarni-25 — APPROVED** [review]

File: Contributing.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 ## ⭐HOW TO MAKE A PULL REQUEST:
 
 **1.** Fork [this](https://github.com/EddieHubCommunity/LinkFree) repository.
-Click on the <a href="https://github.com/EddieHubCommunity/LinkFree"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+Click on the <a href="https://github.com/EddieHubCommunity/LinkFree"><img src="https://img.icons8.com/ios/24/32CD32/code-fork.png"></a> symbol at the top right corner.
```
Comment (Panquesito7) [inline Contributing.md:15]: What about using this link instead? It is the forks page, which makes more sense IMO.
```suggestion
Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/network/members"><img src="https://img.icons8.com/ios/24/32CD32/code-fork.png"></a> symbol at the top right corner.
```

Comment (riadh26) [reply Contributing.md:15]: Yes, i think so 👍

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍
Thanks for reviewing @rohan-kulkarni-25 and @niteshseram! 🙂

**Pull request merged**